nestkernel/nest.h

```diff
@@ -179,6 +179,20 @@ typedef double_t weight;
 typedef long_t delay;
 const long_t delay_max = long_t_max;
 const long_t delay_min = long_t_min;
+
+
+/**
+ * enum type of signal conveyed by spike events of a node
+ * is used upon connect to check if spikes sent by one
+ * neuron are intepreted the same way by receiving neuron
+ */
+enum signal_type
```

Comment:
The enum is a type, so it should be `SignalType`, and the individual enumerators should be treated as constants, i.e., `SPIKE`, `BINARY`, ...